sympy/physics/control/lti.py

```diff
@@ -0,0 +1,655 @@
+
+            if len(other_arg_list) == 0:
+                return Feedback(self, other.args[0])
+            elif len(other_arg_list) == 1:
+                return Feedback(self, *other_arg_list)
+            else:
+                return Feedback(self, Series(*other_arg_list))
+        else:
+            raise ValueError("TransferFunction cannot be divided by {}.".
+                format(type(other)))
+
+    __rtruediv__ = __truediv__
+
+    def __pow__(self, p):
+        p = sympify(p)
+        if not isinstance(p, Integer):
+            raise ValueError("Exponent must be an Integer.")
+        if p == 0:
+            return TransferFunction(1, 1, self.var)
+        elif p > 0:
+            num_, den_ = self.num**p, self.den**p
+        else:
+            p = abs(p)
+            num_, den_ = self.den**p, self.num**p
+
+        return TransferFunction(num_, den_, self.var)
+
+    def __neg__(self):
+        return TransferFunction(-self.num, self.den, self.var)
+
+    @property
+    def is_proper(self):
+        """
+        Returns True if degree of the numerator polynomial is less than
+        or equal to degree of the denominator polynomial, else False.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(b*s**2 + p**2 - a*p + s, b - p**2, s)
+        >>> tf1.is_proper
+        False
+        >>> tf2 = TransferFunction(p**2 - 4*p, p**3 + 3*p + 2, p)
+        >>> tf2.is_proper
+        True
+
+        """
+        return degree(self.num, self.var) <= degree(self.den, self.var)
+
+    @property
+    def is_strictly_proper(self):
+        """
+        Returns True if degree of the numerator polynomial is strictly less
+        than degree of the denominator polynomial, else False.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(a*p**2 + b*s, s - p, s)
+        >>> tf1.is_strictly_proper
+        False
+        >>> tf2 = TransferFunction(s**3 - 2, s**4 + 5*s + 6, s)
+        >>> tf2.is_strictly_proper
+        True
+
+        """
+        return degree(self.num, self.var) < degree(self.den, self.var)
+
+    @property
+    def is_biproper(self):
+        """
+        Returns True if degree of the numerator polynomial is equal to
+        degree of the denominator polynomial, else False.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(a*p**2 + b*s, s - p, s)
+        >>> tf1.is_biproper
+        True
+        >>> tf2 = TransferFunction(p**2, p + a, p)
+        >>> tf2.is_biproper
+        False
+
+        """
+        return degree(self.num, self.var) == degree(self.den, self.var)
+
+
+class Series(Basic):
```

Comment:
This is missing a docstring. All public classes, functions, and methods must have docstrings.

Reply:
Generally, class docstrings should contain the majority of information and examples of how the class should be used, what is its purpose, and where it'll be useful. The function and method docstrings need to explain the purpose of that method, and maybe some examples specfically showing the method's use cases.